test/src/com/connectsdk/service/RokuServiceTest.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * WebOSTVServiceTest
+ * Connect SDK
+ *
+ * Copyright (c) 2015 LG Electronics.
+ * Created by Oleksii Frolov on 16 Jul 2015
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.connectsdk.service;
+
+import com.connectsdk.core.MediaInfo;
+import com.connectsdk.service.capability.MediaPlayer;
+import com.connectsdk.service.command.ServiceCommand;
+import com.connectsdk.service.config.ServiceConfig;
+import com.connectsdk.service.config.ServiceDescription;
+
+import junit.framework.Assert;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+@RunWith(RobolectricTestRunner.class)
+@Config(manifest = Config.NONE)
+public class RokuServiceTest {
+
+    private StubRokuService service;
+    private ServiceDescription serviceDescription;
+
+    class StubRokuService extends RokuService {
+
+        private Object response;
+        private ServiceCommand<?> mLatestCommand;
+
+        public StubRokuService(ServiceDescription serviceDescription, ServiceConfig serviceConfig) {
+            super(serviceDescription, serviceConfig);
+        }
+
+        public void setResponse(Object response) {
+            this.response = response;
+        }
+
+        @Override
+        public void sendCommand(ServiceCommand<?> serviceCommand) {
+            mLatestCommand = serviceCommand;
+            serviceCommand.getResponseListener().onSuccess(response);
+        }
+
+    }
+
+    @Before
+    public void setUp() {
+        serviceDescription = Mockito.mock(ServiceDescription.class);
+        ServiceConfig serviceConfig = Mockito.mock(ServiceConfig.class);
+        service = new StubRokuService(serviceDescription, serviceConfig);
+    }
+
+    @Test
+    public void testServiceNotNull() {
+        Assert.assertNotNull(service);
+    }
+
+    @Test
+    public void testPlayVideo() {
```

Comment:
The `testPlayVideo()` test name is not descriptive, it's not clear what it tests exactly. It should describe the requirement/behavior you're testing, e.g., `testPlayVideoShouldSendHTTPRequestWithCorrectURL()`.